browser-interface/packages/shared/apis/host/runtime7/serialization/crdt/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './deleteComponent'
```

Comment:
cant we use @dcl/sdk here so we avoid duplicating all these files ? 
And use the ones that are already tested ?

Reply:
Absolutely we can, I didn't try to iterate fast, this is not going to be implemented by me, only giving a PoC to show it could work